amethyst_controls/src/bundles.rs

```diff
@@ -8,10 +8,27 @@ use amethyst_core::{
 use super::*;
 
 /// The bundle that creates a flying movement system.
-/// Note: Will not actually create a moving entity. It will only register the needed resources and systems.
-/// The generic parameters A and B are the ones used in InputHandler<A,B>.
-/// You might want to add "fly_movement" and "free_rotation" as dependencies of the TransformSystem.
+///
+/// Note: Will not actually create a moving entity. It will only register the needed resources and
+/// systems. The generic parameters `A` and `B` are the ones used in `InputHandler<A,B>`.
+///
+/// You might want to add `"fly_movement"` and `"free_rotation"` as dependencies of the
+/// `TransformSystem` in order to apply changes made by these systems in the same frame.
 /// Adding this bundle will grab the mouse, hide it and keep it centered.
+///
+/// # Generic parameters
+///
+/// * `A`: This is the key the `InputHandler` is using for axes. Often, this is a `String`.
```

Comment:
Should we encourage the use of `String`?
(I feel this will spawn a doc ticket for "how to make your own input axes / action IDs)